### ISSUE TITLE
Request : Belgium 

### ISSUE THREAD

**Issue by joostschouppe** [pnoll1/osmand_map_creation#62]
Osm data is great here, but address coverage is still missing in a lot of towns. The data is available on OpenAddresses.

(also, awesome project!)

**Comment by pnoll1** [comment]
Build is up: https://opensupermaps.com/static/downloads/Be.obf. 

This is the first area I've seen with address files named like this. Not sure what the DE, FR or NL stand for exactly. Addreses might be in the file several times, in several languages.

- bosa-region-wallonia-de-addresses-state.geojson        

- bosa-region-wallonia-fr-addresses-state.geojson

Let me know if things look strange.

**Comment by joostschouppe** [comment]
Thank you, I'll check it out.
The NL, FR and DE are language codes. Brussels is officially bilingual, so both Dutch (NL) and French make sense. For Wallonia there is a small region that is German (DE) speaking. Looking at file sizes, it seems they included the whole region in the German language file.

**Comment by pnoll1** [comment]
Build has been uploaded to 0.97 release and BE added to build roster. Closing this.

I opened a separate issue to look into the data. #64

**Comment by joostschouppe** [comment]
Tested:
- It works fine in Brussels and Flanders
- it does not work in Wallonia: if you look for any town (try Arlon or Sankt Vith), it will first pop up all the housenumbers without any street name, and only after it shows the actual streets and their housenumbers. So a lot of duplicated, empty data is there. It might just work if you drop the German language file.

**Comment by pnoll1** [comment]
alright, this took a while to unravel since BE already has full address coverage with duplicates in the OSM data.

I thought OsmAnd would compensate for the missing street fields by matching the addresses to the road network. That did not happen so the data is useless. I'll add a filter to get rid of all addresses with a missing street field.

**Comment by pnoll1** [comment]
Code was broken when I built these files. Most of the data from OA didn't make it into file.

Fixed files coming with todays release.

**Comment by joostschouppe** [comment]
Thanks, I tested and seems to work just fine!
